tests/MSBuildDeviceIntegration/Tests/XASdkDeployTests.cs

```diff
@@ -13,7 +13,7 @@
 namespace Xamarin.Android.Build.Tests
 {
 	[NonParallelizable]
-	[Category ("UsesDevices"), Category ("DotNetIgnore")] // These don't need to run under `--params dotnet=true`
+	[Category ("UsesDevices"), Category ("SmokeTests"), Category ("DotNetIgnore")] // These don't need to run under `--params dotnet=true`
```

Comment:
If you make any other changes here you might update the comment:
```suggestion
	[Category ("UsesDevices"), Category ("SmokeTests"), Category ("DotNetIgnore")] // These don't need to run under `dotnet test`
```
But it's fine to leave it if this is finished.